sunpy/map/mapbase.py

```diff
@@ -365,6 +368,21 @@ def center(self):
                                    self.reference_coordinate.y))
 
     @property
+    def shift(self):
+        """Returns the shift added to the map center."""
```

Comment:
This could also do with a more descriptive docstring. This might be the docstring people first encounter when discovering this functionality.
Also "returns" is incorrect terminology for a property, because it presents as an attribute.